shared_infra/dynamo_write_heartbeat/src/test_dynamo_write_heartbeat.py

```diff
@@ -0,0 +1,50 @@
+# -*- encoding: utf-8 -*-
+
+import pytest
+import boto3
+import os
+import dynamo_write_heartbeat
+
+TABLE_NAMES = 'Test1, Test2'
+ENDPOINT_URL = 'http://localhost:8000'
+
+
+@pytest.yield_fixture(autouse=True)
+def run_around_tests():
+    dynamodb = boto3.client('dynamodb', endpoint_url=ENDPOINT_URL)
```

Comment:
Nice use of fixtures. 👍 

We have a pytest fixture in `shared_conftest.py` in the root of the repo, available to all Lambda tests.

If you make `dynamodb_client` an argument to this function, it gets filled in by py.test with appropriate auth/endpoint URL. Ditto for SQS/SNS/S3.